epoxy-compose/src/main/java/com/airbnb/epoxy/ComposeInterop.kt

```diff
@@ -0,0 +1,51 @@
+package com.airbnb.epoxy
+
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.viewinterop.AndroidView
+
+class ComposeEpoxyModel(
+    private val composeFunction: @Composable () -> Unit
+) : EpoxyModelWithView<ComposeView>() {
+    override fun buildView(parent: ViewGroup): ComposeView = ComposeView(parent.context)
+
+    override fun bind(view: ComposeView) {
+        super.bind(view)
+        view.setContent(composeFunction)
+    }
+}
+
+fun EpoxyController.composableInterop(
```

Comment:
These will need good documentation for public usage. do I need to do that as a follow up?

Reply:
I can put something up as a follow up